src/commands/appSettings/connectionSettings/azureWebJobsStorage/validateStorageConnection.ts

```diff
@@ -4,29 +4,24 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { AzureWizard } from "@microsoft/vscode-azext-utils";
-import { CodeAction, ConnectionKey, localStorageEmulatorConnectionString } from "../../../../constants";
+import { CodeAction, ConnectionKey } from "../../../../constants";
 import { getLocalSettingsConnectionString } from "../../../../funcConfig/local.settings";
 import { IConnectionPromptOptions } from "../IConnectionPromptOptions";
 import { ISetConnectionSettingContext } from "../ISetConnectionSettingContext";
 import { AzureWebJobsStorageExecuteStep } from "./AzureWebJobsStorageExecuteStep";
 import { AzureWebJobsStoragePromptStep } from "./AzureWebJobsStoragePromptStep";
 import { IAzureWebJobsStorageWizardContext } from "./IAzureWebJobsStorageWizardContext";
 
-// Supports validation on both 'debug' and 'deploy'
 export async function validateStorageConnection(context: Omit<ISetConnectionSettingContext, 'projectPath'>, projectPath: string, options?: IConnectionPromptOptions): Promise<void> {
+    if (context.action === CodeAction.Deploy) {
```

Comment:
Do we need even need this?` validateStorageConnection` never gets called in the deploy scenario anyway, right? I think it's only in pre-debugging.

Reply:
Correct, I wanted to add it in there so that it's extremely explicit, especially since the rest of the connection type validations can happen in both debug or deploy